src/confcom/azext_confcom/template_util.py

```diff
@@ -341,14 +378,38 @@ def change_key_names(dictionary) -> Dict:
     return dictionary
 
 
+def replace_params_and_vars(params: dict, vars_dict: dict, attribute):
+    out = None
+    if isinstance(attribute, (int, float, bool)):
+        out = attribute
+    elif isinstance(attribute, str):
+        out = find_value_in_params_and_vars(params, vars_dict, attribute)
+        # TODO: this can be optimized to not have so many groups in the single match
+        regex = r"(\s*\[\s*(parameters|variables))(\(\s*'([^\.\/]+?)'\s*\)\])"
```

Comment:
Since you made a global constant out of the other regex, I would do the same here. It will make updating these regexes easier in the future.